logging.rst

```diff
@@ -74,7 +74,7 @@ to write logs using the :phpfunction:`syslog` function:
 
     .. code-block:: yaml
 
-        # config/packages/monolog.yaml
+        # config/packages/prod/monolog.yaml
```

Comment:
What about the other formats below?

Reply:
I would not add them to the dev file I think. It could be too confusing if you do not see all logs anymore during development. But maybe we should add a warning?